.github/PULL_REQUEST_TEMPLATE

```diff
@@ -19,9 +19,9 @@ Please describe how you tested this change and how a reviewer could reproduce yo
 # Checklist
 
 <!--
-Please check the appropriate items below if they apply to your diff. This is required for changes to Expo modules.
+Please check the appropriate items below if they apply to your diff.
 -->
 
-- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
-- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
+- [ ] I added a `changelog.md` entry and rebuilt the sources according to [this short guide](https://github.com/expo/expo/blob/main/CONTRIBUTING.md#-before-submitting)
```

Comment:
```suggestion
- [ ] Added a `changelog.md` entry and rebuilt the package sources according to [this short guide](https://github.com/expo/expo/blob/main/CONTRIBUTING.md#-before-submitting)
```

Feels odd to specify a personal pronoun for one item but not for the other two. I think a good pattern would be to either use it for all three or remove it from here.

I also think we can add clarity about what "sources" refers to? I think in almost all cases, they are package sources.